app/services/adapters/structs/crime_application.rb

```diff
@@ -73,6 +77,8 @@ def outgoings_payments
       end
 
       def capital
+        return nil unless means_details
```

Comment:
You could have means details without capital so this is condition right? Same for outgoings